<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRegExpEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="regexpEdit">
        <property name="styleSheet">
         <string notr="true">background: #ff0000;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findbutton">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_dstEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>PatternSyntax:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_patternSyntax">
            <item>
             <property name="text">
              <string>RegExp</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wildcard</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FixedString</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RegExp2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>WildcardUnix</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>W3CXmlSchema11</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>58</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="m_eachline">
            <property name="text">
             <string>Apply on each line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_exactMatch">
            <property name="text">
             <string>exactMatch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_skipEmpty">
            <property name="text">
             <string>skip Empty</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_displayNotMatched">
            <property name="text">
             <string>displayNotmatched</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_showAllMatch">
            <property name="text">
             <string>Show All Match</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>98</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="m_caseSensitive">
          <property name="text">
           <string>Case Sensitive</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_minimal">
          <property name="text">
           <string>Minimal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="before"/>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="after">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
